Lib/ufo2ft/featureWriters/cursFeatureWriter.py

```diff
@@ -0,0 +1,129 @@
+from fontTools.misc.fixedTools import otRound
+
+from ufo2ft.featureWriters import BaseFeatureWriter, ast
+from ufo2ft.featureWriters.kernFeatureWriter import unicodeScriptDirection
```

Comment:
maybe unicodeScriptDirection should be moved to ufo2ft.util if it's useful for other writers than kern